urllib3/poolmanager.py

```diff
@@ -142,7 +137,7 @@ def connection_from_url(self, url):
         u = parse_url(url)
         return self.connection_from_host(u.host, port=u.port, scheme=u.scheme)
 
-    def urlopen(self, method, url, redirect=True, **kw):
+    def urlopen(self, method, url, redirect=True, retries=None, **kw):
```

Comment:
I think it's fine too, just wanted to check we did it on purpose and that it wasn't left in from a previous round.